homeassistant/components/sensor/recollect_waste.py

```diff
@@ -0,0 +1,85 @@
+"""
+Support for Recollect Waste curbside collection pickup.
+
+For more details about this platform, please refer to the documentation at
+https://github.com/stealthhacker/python-recollect-waste
```

Comment:
This should be a link to the Hone Assistant documentation, not the library documentation.